test/BuyTshirt.spec.ts

```diff
@@ -1,49 +1,66 @@
-import { browser, protractor} from 'protractor';
+import { browser, protractor } from 'protractor';
 import { MenuContentPage, ProductAddedModalPage, ProductDetailPage,
    SummaryStepPage, SignInStepPage, AddressStepPage, ShippingStepPage,
    PaymentStepPage, BankPaymentPage, OrderResumePage } from '../src/page';
 
-describe('Buy a t-shirt', () => {
-  const menuContentPage: MenuContentPage = new MenuContentPage();
-  const productDetailPage: ProductDetailPage = new ProductDetailPage();
-  const productAddedModalPage: ProductAddedModalPage = new ProductAddedModalPage();
-  const summaryStepPage: SummaryStepPage = new SummaryStepPage();
-  const signInStepPage: SignInStepPage = new SignInStepPage();
-  const addressStepPage: AddressStepPage = new AddressStepPage();
-  const shippingStepPage: ShippingStepPage = new ShippingStepPage();
-  const paymentStepPage: PaymentStepPage = new PaymentStepPage();
-  const bankPaymentPage: BankPaymentPage = new BankPaymentPage();
-  const orderResumePage: OrderResumePage = new OrderResumePage();
-
-  it('then should be bought a t-shirt', async () => {
+describe('Abrir la página en el navegador', () => {
+  it('entonces debería abrirse la página', async () => {
     await browser.get('http://automationpractice.com/');
 
-    await menuContentPage.goToTShirtMenu();
+  });
+
+  describe('Proceso de compra de la camiseta', () => {
+    const menuContentPage: MenuContentPage = new MenuContentPage();
+    const productDetailPage: ProductDetailPage = new ProductDetailPage();
+    const productAddedModalPage: ProductAddedModalPage = new ProductAddedModalPage();
+    const summaryStepPage: SummaryStepPage = new SummaryStepPage();
+    it('entonces se debería comprar la camiseta', async () => {
 
-    await productDetailPage.goToProductDetail();
+      await menuContentPage.goToTShirtMenu();
+      await productDetailPage.goToProductDetail();
+      await productAddedModalPage.goToProductAddedModal();
 
-    await productAddedModalPage.goToProductAddedModal();
+      const expectedCondition = protractor.ExpectedConditions;
+      const condition = expectedCondition
+            .elementToBeClickable(summaryStepPage.getSummaryStepPage);
+      await browser.wait(condition, 3000, 'error waiting');
+      await summaryStepPage.goToSummaryStep();
 
-    var expectedCondition = protractor.ExpectedConditions;
-    var condition= expectedCondition.elementToBeClickable(summaryStepPage.getSummaryStepPage);
-    await browser.wait(condition, 3000, 'error waiting');
-    await summaryStepPage.goToSummaryStep();
+    });
 
-    await signInStepPage.goToSignInStep();
+    describe('Logeo en la aplicación', () => {
+      const signInStepPage: SignInStepPage = new SignInStepPage();
+      const addressStepPage: AddressStepPage = new AddressStepPage();
+      it('entonces debería estar logeado en la aplicación', async () => {
```

Comment:
los "it" no deben ser usados así, el "it" siempre será el que contenga la verificación del test.